### DISCUSSION TITLE
Unchanged string change saved as translation

### DISCUSSION THREAD

**Discussion by comradekingu** [WeblateOrg/weblate#4877] [Q&A]
https://hosted.weblate.org/translate/weblate/application/nb_NO/?checksum=a48099b72a1f28c8

How is this possible? Can it be avoided, or reverted?

**Comment by nijel** [comment]
How could Weblate know whether it needs a change? I'd assume that Kurt considered that as a good translation...

**Comment by comradekingu** [comment]
@nijel Actually what happens is that reverting a string saves I think the version before it or the current version, with "0 characters edited". Just noticed when trying to revert some errors he reinstated.

For example https://hosted.weblate.org/translate/weblate/application/nb_NO/?checksum=469af9ca7ace5891

I think he meant to revert this
https://hosted.weblate.org/translate/weblate/application/nb_NO/?checksum=d5c04a87992d2f0f

Edit: I have to click the one above to revert to the string below.

Also #4889 is a problem that is possibly related. All of these are opened from Kurts string history

Edit: I edited this to have the dash, and now it doesn't https://hosted.weblate.org/translate/weblate/application/nb_NO/?checksum=8a0b862b87b18b0
However there is no history to suggest it changed to not have the dash.

**Reply by nijel** [reply]
The history shows the dash adding and removal, The tricky thing is that strike-through dash is not really distinguishable from strike-through space...
